spec/api/v1/controllers/api_crops_controller_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper'
+
+describe Api::V1::CropsController do
+
+  let(:crop) { FactoryGirl.create(:tomato) }
+
+  def check_license(response)
+    json = JSON.parse response.body
+    json['license'].should_not == nil
+    json['license']['name'].should == "Creative Commons Attribution ShareAlike 3.0 Unported"
+    json['license']['short_name'].should == "CC-BY-SA"
+    json['license']['url'].should == "http://creativecommons.org/licenses/by-sa/3.0/"
+    json['license']['credit'].should == "Growstuff"
+    json['license']['link'].should == "http://growstuff.org/"
+    json['license']['easy_link'].should == '<a href="http://creativecommons.org/licenses/by-sa/3.0/">CC-BY-SA</a> <a href="http://growstuff.org/">Growstuff</a>'
```

Comment:
Can this be written like so? 

`expect(json['license']).to_not eq(nil)`

(Otherwise the tests are looking great!)